RecoEcal/EgammaClusterAlgos/src/PFECALSuperClusterAlgo.cc

```diff
@@ -145,6 +152,14 @@ void PFECALSuperClusterAlgo::update(const edm::EventSetup& setup) {
   channelStatus_ = esChannelStatusHandle_.product();
 }
 
+void PFECALSuperClusterAlgo::updateSCParams(const edm::EventSetup& setup) {
+  edm::ESHandle<EcalMustacheSCParameters> ecalMustacheSCParamsHandle_ = setup.getHandle(ecalMustacheSCParametersToken_);
+  mustacheSCParams_ = ecalMustacheSCParamsHandle_.product();
```

Comment:
These could be simplified to
```suggestion
  mustacheSCParams_ = &setup.getData(ecalMustacheSCParametersToken_);
```